serialization-jackson/src/main/resources/reference.conf

```diff
@@ -92,6 +92,16 @@ pekko.serialization.jackson {
     FAIL_ON_UNKNOWN_PROPERTIES = off
   }
 
+  # Configuration of the ObjectMapper deserialization features.
+  # See com.fasterxml.jackson.databind.cfg.EnumFeature
+  # Enum values corresponding to the EnumFeature and their
+  # boolean values, for example:
+  #
+  # enum-features {
+  #   READ_ENUM_KEYS_USING_INDEX = off
+  # }
+  enum-features {}
```

Comment:
Should we include a Jackson version required in the comment?

Reply:
thing is that this EnumFeature class exists since Jackson 2.14 and even Pekko 1.0 requires Jackson 2.14